packages/rs-platform-version/src/version/dpp_versions/dpp_state_transition_versions/v3.rs

```diff
@@ -0,0 +1,46 @@
+use crate::version::dpp_versions::dpp_state_transition_versions::{
+    AddressFundsTransitionVersions, ContractTransitionVersions, DPPStateTransitionVersions,
+    DocumentTransitionVersions, DocumentsBatchTransitionValidationVersions,
+    DocumentsBatchTransitionVersions, IdentityCreditWithdrawalTransitionVersions,
+    IdentityTransitionAssetLockVersions, IdentityTransitionVersions,
+};
+
+pub const STATE_TRANSITION_VERSIONS_V3: DPPStateTransitionVersions = DPPStateTransitionVersions {
+    documents: DocumentTransitionVersions {
+        documents_batch_transition: DocumentsBatchTransitionVersions {
+            validation: DocumentsBatchTransitionValidationVersions {
+                find_duplicates_by_id: 0,
+                validate_base_structure: 0,
+            },
+        },
+    },
+    identities: IdentityTransitionVersions {
+        max_public_keys_in_creation: 6,
+        asset_locks: IdentityTransitionAssetLockVersions {
+            required_asset_lock_duff_balance_for_processing_start_for_identity_create: 200000,
+            required_asset_lock_duff_balance_for_processing_start_for_identity_top_up: 50000,
+            required_asset_lock_duff_balance_for_processing_start_for_address_funding: 50000,
+            validate_asset_lock_transaction_structure: 0,
+            validate_instant_asset_lock_proof_structure: 0,
+        },
+        credit_withdrawal: IdentityCreditWithdrawalTransitionVersions {
+            default_constructor: 1,
+        },
+        calculate_min_required_fee_on_identity_create_transition: 1,
```

Comment:
Please document changes in versions, now it's quite hard to read and understand the changes.

For example:
```suggestion
        /// Changed from 0 to 1 in v3
        calculate_min_required_fee_on_identity_create_transition: 1,
```